api/types/types.go

```diff
@@ -189,15 +208,15 @@ type Info struct {
 	NCPU               int
 	MemTotal           int64
 	DockerRootDir      string
-	HttpProxy          string
-	HttpsProxy         string
+	HTTPProxy          string
```

Comment:
Same as above, as it is serialized as JSON, I think it would be better to do something like :

```go
HTTPProxy string `json:"HttpProxy"`
HTTPSProxy string `json:"HttpsProxy"`
```